fr.handbrake.ghb.Plugin.dvdcss.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "fr.handbrake.ghb.Plugin.dvdcss",
+    "runtime": "fr.handbrake.ghb",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//46",
+    "build-extension": true,
+    "separate-locales": false,
+    "build-options": {
+        "prefix": "/app/extensions/dvdcss"
+    },
+    "modules": [
+        {
+            "name": "metadata",
```

Comment:
this doesn't need to be a separate module

Reply:
I've tried merging them into a single module, but I've hit some issues.

If I create a single module with the union of the sources lists, I have to decide which buildsystem to use.

By default, the `libdvdcss` module appears to automatically build with autotools as required.

When combining the two modules, if I remove the `buildsystem: simple` directive, then the `metainfo.xml` file fails to install, but the `libdvdcss` library builds and installs correctly.

But if I include `buildsystem: simple` and `build-commands`, then `metainfo.xml` installs correctly, but it no longer automatically builds `libdvdcss` with autotools.

I thought perhaps I could just manually add all the required build commands to the `build-commands` list. However, it's not clear to me that this is any better than having two modules.
Also, it seems to build in a slightly different way, and produces extra files which weren't included before.

Could you provide some direction here?

Reply:
merge the sources and add `install -Dm0644 ...` to `post-install` of the other module.

Reply:
Thanks!